service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -117,6 +133,23 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
     logger.info(RESOURCE_CREATED, getResourceType(), aks.id(), getMRGName(context));
   }
 
+  /** see https://github.com/Azure/azure-sdk-for-java/issues/31271 */
+  private static void enableWorkloadIdentity(KubernetesCluster aks) {
```

Comment:
So glad this works -- I would like Leo to move to use Workload Identity as well. Referencing IA ticket: https://broadworkbench.atlassian.net/browse/IA-3804

Reply:
I don't think Leo will need to enable anything on the cluster but a k8s service account will need to be created as well as a federated identity on the UAMI